cmd/emp/run.go

```diff
@@ -33,6 +33,8 @@ Run a process on Heroku. Flags such as` + " `-a` " + `may be parsed out of
 the command unless the command is quoted or provided after a
 double-dash (--).
 
+You may need to press enter after this command to continue.
```

Comment:
Can we add a little more information to this? Right now, it's not clear _why_ you would need to do this. Maybe something like:

```
When running an attached process that reads from stdin (like bash) you may experience a "hang". Usually, pressing a key like "enter" will flush the output to your terminal. See https://github.com/remind101/empire/issues/609
```